app/src/layouts/Layout.tsx

```diff
@@ -11,13 +11,20 @@ type LayoutProps = {
   children: ReactNode;
   rest?: unknown;
 };
+
+const pageTitle = atom("LinkUp");
+export const updateTitle = atom(null, (get, set, newTitle:string) =>
+  set(pageTitle, newTitle)
+);
+
 function Layout({ children }: LayoutProps) {
   const [isMenuOpen] = useAtom(menuOpen);
+  const title = useAtomValue(pageTitle);
```

Comment:
also here you can just do
```suggestion
  const [title] = useAtom(pageTitle);
```

so it saves from using another import